recipe/meta.yaml

```
@@ -284,6 +284,27 @@ outputs:
         - osgeo._osr
       commands:
         - python run_test.py
+        # Check .py GDAL utilities
+        # --help not implemented as documented for gdal_merge.py, gdal_retile.py, gdal_sieve.py, gdalmove.py, gdal_edit.py, gdal_pansharpen.py, gdalcompare.py, ogrmerge.py?
+        # also, --version does not work with e.g. rgb2pct.py...
+        - rgb2pct.py --help
+        - pct2rgb.py --help
+        - gdalattachpct.py --help
```

Getting an error on this one:
```
gdalattachpct.py --help
Usage: gdalattachpct.py <pctfile> <infile> <outfile>
```
We might have to change this to:
```suggestion
        - if not exist %PREFIX%\\Library\\bin\\gdalattachpct.py exit 1  # [win]
        - test -f $PREFIX/bin/gdalattachpct.py  # [unix]
```
though I'm certainly up for smarter suggestions if anyone has one.

---

Done (it wasn't entirely clear to me in local testing that it was an error). The better suggestion, I guess, would be to properly fix `--help` in upstream GDAL for this and other scripts.

---

It would be even better to make them entry points instead of scripts so they work as expected in Windows as discussed in #834